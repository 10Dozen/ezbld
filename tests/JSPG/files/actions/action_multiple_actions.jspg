@ Action One
*scene: SceneName
This is action one content.

@ Action Two
*scene: SceneName
This is action two content.

@ Action Three
*scene: SceneName
This is action three content.