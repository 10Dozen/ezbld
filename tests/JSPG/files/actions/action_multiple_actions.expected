[0]
cls = JSPGAction
name = Action One | TEXT
scene = SceneName | TEXT
desc = "This is action one content."

[1]
cls = JSPGAction
name = Action Two | TEXT
scene = SceneName | TEXT
desc = "This is action two content."

[2]
cls = JSPGAction
name = Action Three | TEXT
scene = SceneName | TEXT
desc = "This is action three content."